### PR TITLE
Added API spec

### PR DESCRIPTION
**Pull request by shubhaangi04**

### CODE REVIEW

File: Product Management Service/Documents/API_Spec.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+GET /products  - Get all products
+
+GET /products/{ID}  - Get product by ID
```
Comment (thisissvikas) [inline Product Management Service/Documents/API_Spec.txt]: make id small

File: Product Management Service/Documents/API_Spec.txt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    price : 50,
+    category : "Bath & Body",
+    description : "Use for effective cleansing of hands and body, without losing the skin's natural moisture.",
+    specifications : {
+        brand : "Nivea"
+        weight : 125
+        colour : "white"
+    },
+    created_timestamp : 2021-10-18 16:00:00,
+    updated_timestamp : 2021-10-18 16:00:00
```
Comment (thisissvikas) [inline Product Management Service/Documents/API_Spec.txt]: this should be system generated

File: Product Management Service/Documents/API_Spec.txt [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    price : 50,
+    category : "Cleansers",
+    description : "Use for effective cleansing of hands and body, without losing the skin's natural moisture.",
+    specifications : {
+        brand : "Nivea"
+        weight : 125
+        colour : "white"
+    },
+    created_timestamp : 2021-10-18 16:00:00,
+    updated_timestamp : 2021-10-18 16:00:00
```
Comment (thisissvikas) [inline Product Management Service/Documents/API_Spec.txt]: we should not be allowing this update

**Review by thisissvikas — CHANGES_REQUESTED** [review]
I also see the possibility of few more endpoints that might be needed. Such as:
- Endpoint to get all products of a category
- Endpoint to get all the products of a particular brand
- Endpoint to delete all products

Even after these, I see further addition to these as per requirement, but these are for sure I see as necessary.

**Review by thisissvikas — APPROVED** [review]

**Comment by shubhaangi04** [comment]
Related to : https://github.com/krvikas1011/shop-keeper/issues/5

**Pull request merged**